config/locales/diaspora/en.yml

```diff
@@ -117,6 +117,20 @@ en:
       account_closing_scheduled: "The account of %{name} is scheduled to be closed. It will be processed in a few moments..."
       account_locking_scheduled: "The account of %{name} is scheduled to be locked. It will be processed in a few moments..."
       account_unlocking_scheduled: "The account of %{name} is scheduled to be unlocked. It will be processed in a few moments..."
+      make_admin: "Make Admin"
+      remove_admin: "Remove Admin"
+      make_moderator: "Make Moderator"
+      make_spotlight: "Make Spotlight"
+      remove_spotlight: "Remove Spotlight"
+      remove_moderator: "Remove Moderator"
+      add_admin: "The account of %{name} has been made an Admin."
+      delete_admin: "The account of %{name} has been removed as an Admin."
+      add_moderator: "The account of %{name} has been made an Moderator."
+      delete_moderator: "The account of %{name} has been removed as an Moderator."
+      add_spotlight: "The account of %{name} has been made an Spotlight."
+      delete_spotlight: "The account of %{name} has been removed as an Spotlight."
+      does_not_exist: "User does not exist!"
+      role_implemented: "The role of %{name} has already been implemented!"
```

Comment:
Wording seems improvable :P @goobertron any idea? It's meant to say the role was already set or removed.

Reply:
Sorry, I didn't receive a notification for this mention.

There are some grammatical errors here; also, all text should respect normal capitalisation guidelines for English sentences (not Each Word Capitalised). And I don't think 'The account of %{name}' is necessary; '%{name}' alone is enough. (And `remove_moderator` should ideally be moved to under `make_moderator`.) They could all use some polishing, and I would word them thus:

      make_admin: "Make admin"
      remove_admin: "Remove as admin"
      make_moderator: "Make moderator"
      remove_moderator: "Remove as moderator"
      make_spotlight: "Add to community spotlight"
      remove_spotlight: "Remove from community spotlight"
      add_admin: "%{name} has been made an admin."
      delete_admin: "%{name} has been removed from the list of admins."
      add_moderator: "%{name} has been made a moderator."
      delete_moderator: "%{name} has been removed from the list of moderators."
      add_spotlight: "%{name} has been added to community spotlight."
      delete_spotlight: "%{name} has been removed from community spotlight."
      does_not_exist: "User does not exist!"

The final one is tricky. Is it to be used both when a role is added and removed, or only when it has been added? If the latter, I would say:

      role_implemented: "%{name} has already been given this role!"

if the former, you'll either need two lines:

      role_implemented: "%{name} has already been given this role!"
      role_removal_implemented: "%{name} has already been removed from this role!"

or try to find some better wording that will fit both adding and removal cases. Let me know if that's needed.